app/scripts/controllers/TimerCtrl.js

```diff
@@ -1,12 +1,14 @@
 (function() {
 
-    function TimerCtrl($scope, $interval, CLOCK_TIMES) {
+    function TimerCtrl($scope, $interval, CLOCK_TIMES, Task) {
       $scope.buttonMsg = "Start a Session";
       $scope.breakButtonMsg = "Start a Break";
       //$scope.time = 1500;
       $scope.time = CLOCK_TIMES.totalWorkTime;
       $scope.onBreak = false;
       $scope.format = 'mm:ss';
+      $scope.task = '';
+      $scope.tasks = Task.all;
 
       console.log("TimerCtrl Hears you!")
```

Comment:
I would say remove this but I kind of love it.

Reply:
remove the `console.log("TimerCtrl Hears you!")`?  Over my dead body!!!